legendary_eureka.py

```diff
@@ -34,6 +39,99 @@ class Glassdoor(Resource):
     def __init__(self):
         Resource.__init__(self, 'glassdoor')
 
-    # TODO
-    def get_jobs(self):
-        pass
+    # TODO: wrap requests.post parts with try / except.
+    def get_jobs(self, job_title: str, job_location: str = None, job_type: str = ''):
+        """
+        using part of code from: https://gist.github.com/scrapehero/352286d0f9dee87990cd45c3f979e7cb
+        :param job_title: for example: 'python developer'
+        :param job_type: optional, 'fulltime' or ___ or ___ TODO: get job_type possibilities
+        :param job_location: Jerusalem (Israel), Tel Aviv-Yafo (Israel), Haifa (Israel), Ashdod (Israel),
+         Bnei Brak (Israel), Netanya (Israel), Bene Beraq (Israel), Bat Yam (Israel), Ramat Gan(Israel),
+         Ashqelon (Israel).
```

Comment:
Can't we have `any`?

Reply:
I didn't find an option to search in 'Israel' , so by default if you don't set 'job_location' function parameter you will get all the jobs lists one after another for these locations.

Reply:
Got list of all cities, by default job location is None so it will show general israel jobs.

Reply:
We want to present jobs in all of the cities in Israel by default